recipes/ncurses/all/conanfile.py

```diff
@@ -1,31 +1,39 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
-import contextlib
-import functools
 import os
-import textwrap
 
-required_conan_version = ">=1.33.0"
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.build import cross_building, stdcpp_library
+from conan.tools.env import Environment
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get
+from conan.tools.gnu import Autotools, AutotoolsToolchain, PkgConfigDeps
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime, unix_path
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.53.0"
 
 
 class NCursesConan(ConanFile):
     name = "ncurses"
     description = "The ncurses (new curses) library is a free software emulation of curses in System V Release 4.0 (SVr4), and more"
-    topics = ("ncurses", "terminal", "screen", "tui")
+    license = "X11"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://www.gnu.org/software/ncurses"
-    license = "X11"
-    settings = "os", "compiler", "build_type", "arch"
+    topics = ("terminal", "screen", "tui")
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
         "with_widec": [True, False],
         "with_extended_colors": [True, False],
         "with_cxx": [True, False],
         "with_progs": [True, False],
-        "with_ticlib": ["auto", True, False],
+        "with_ticlib": [True, False],
         "with_reentrant": [True, False],
-        "with_tinfo": ["auto", True, False],
+        "with_tinfo": [True, False],
```

Comment:
Please, inform how do you change these values in term of always True, but considering that the system will actually do a check to enable/disable first.

Reply:
I restored the "auto" default value for clarity and backwards compatibility.